worker/src/unstract/worker/clients/test_docker.py

```diff
@@ -127,14 +127,16 @@ def test_get_container_run_config(docker_client, mocker):
         envs={"KEY": "VALUE"},
         auto_remove=True,
     )
-
+    execution_data_folder_prefix = os.getenv(Env.EXECUTION_RUN_DATA_FOLDER_PREFIX)
     mocker_normalize.assert_called_once_with("test-image")
     assert config["name"] == "test-image"
     assert config["image"] == "test-image:latest"
     assert config["command"] == ["echo", "hello"]
     assert config["environment"] == {
         "KEY": "VALUE",
-        "EXECUTION_RUN_DATA_FOLDER": "/source/org123/wf123/ex123",
+        "EXECUTION_RUN_DATA_FOLDER": (
+            f"{execution_data_folder_prefix}" "/org123/wf123/ex123"
```

Comment:
@Deepak-Kesavan we usually need to avoid calculated assertions when ever possible. That is the better way of handling this. So what I would suggest is for this case set an env `EXECUTION_RUN_DATA_FOLDER_PREFIX` at the beginning of this test. Then we can simply hardcode that value here.

Reply:
@ritwik-g Should I move it to the top or is it fine if it is kept at L130?

Reply:
You can keep under this line. This needs to happen before function calls. https://github.com/Zipstack/unstract/pull/616/files#diff-b89214285c8ac571e7607a2323f9d6c34d9d5bad91dd6dbb2a1b2a70496d69a4R111